### PR TITLE
Refactor-droid

### PR DESCRIPTION
**Pull request by SimenIversen**
task done

### CODE REVIEW

File: 1_space_odyssey/js/simen-9000/app.js [hunk 1]
```diff
@@ -100,11 +115,43 @@
...
+      },
+      superDroid:{
+        hasWeapon: true,
+        isInGoodMood: true,
+      },
+    }
     console.log(".-.-. Booting .-.-.");
-    console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+    console.log("head",
```
Comment (mannuelf) [inline 1_space_odyssey/js/simen-9000/app.js:130]: cool

File: 1_space_odyssey/js/simen-9000/app.js [hunk 2]
```diff
@@ -100,11 +115,43 @@
     var canRun = true;
     var canFly  = false;
     var hasWeapon = true;
-    var isInGoodMood = true;
-
-
+    var isInGoodMood = true; */
+
+    var simensAndoid = {
+      head: {
```
Comment (mannuelf) [inline 1_space_odyssey/js/simen-9000/app.js:104]: cool

File: 1_space_odyssey/js/simen-9000/app.js [hunk 3]
```diff
@@ -85,7 +85,22 @@
...
+ - Feel free to create any new properties that will further describe your android [ anyThingYouCanDreamOf ].
+ - next update your application to use this newly refactored code
+
+ tip: the console.log that prints the variables of the android will need to be updated
+
+ Level 2::::::::::::::
+ Refactor the planets array to be an array of objects, each object must be its own planet.
+  - next update your application to use this newly refactored code
+*/
+    /* var isAlive = true;
```
Comment (mannuelf) [inline 1_space_odyssey/js/simen-9000/app.js]: you can delete thes commented out variables now. you have successfully refactored your code and this is DEAD code now.

**Review by mannuelf — CHANGES_REQUESTED** [review]

**Comment by SimenIversen** [comment]
The red marking that you have marked is nowhere in my code.. cant find the anymore

**Pull request merged**